examples/src/main/java/com/mapbox/navigation/examples/core/FreeDriveNavigationActivity.kt

```diff
@@ -63,7 +60,10 @@ class FreeDriveNavigationActivity : AppCompatActivity(), OnMapReadyCallback {
                 this,
                 Utils.getMapboxAccessToken(this)
         )
-        val updatedOptions = mapboxNavigationOptions.toBuilder().onboardRouterConfig(getOnBoardRouterConfig()).build()
+
+        val updatedOptions = mapboxNavigationOptions.toBuilder()
+            .onboardRouterConfig(OnboardRouterOptions.createDefaultOptions(this))
+            .build()
```

Comment:
This seems unnecessary as we're getting the default options already from https://github.com/mapbox/mapbox-navigation-android/blob/ccdc1980efa6fbeae0c034311f7f27124ccba4e3/examples/src/main/java/com/mapbox/navigation/examples/core/FreeDriveNavigationActivity.kt#L59-L62

Note that the previous onboard config wasn't the default.

Reply:
Good catch again. Also the non-default example is in place. Here is how to override the tile uri, and the version.

```kotlin
val updatedOptions = mapboxNavigationOptions.toBuilder()
    .onboardRouterConfig(OnboardRouterOptions.defaultOptions(this)
        .endpoint(OnboardRouterEndpointOptions.Builder()
            .tilesUri("https://api-routing-tiles-staging.tilestream.net")
            .version("2020_02_02-03_00_00")
            .build()
        ).build())
    .build()
```